server/db/postgres.js

```diff
@@ -33,6 +33,14 @@ var postgres = {
       output = {};
       Object.keys(row).forEach(function(k) {
         output[camelize(k)] = row[k];
+        // special case: json extracted_data blob
+        // rather than use recursion, take advantage of the fact that all underscored keys
+        // are top-level and just nest another function. stupid, simple, works.
+        if (k === 'extracted_data') {
+          Object.keys(row.extracted_data).forEach(function(key) {
+            output.extractedData[camelize(key)] = row.extracted_data[key];
+          });
+        }
```

Comment:
i'm a bit unclear here... We aren't using `k` in the inner `forEach()` or anything, should we just move this whole block outside of the outer `Object.keys(row).forEach()` call so we don't check the `if` statement each time?